colorizer_data/converter.py

```diff
@@ -190,40 +214,55 @@ def _write_features(
 def _should_regenerate_frames(
     writer: ColorizerDatasetWriter, data: DataFrame, config: ConverterConfig
 ) -> bool:
-    # get object count and regenerate frames if it has changed
-    num_objects = data[config["object_id_column"]].nunique(False)
 
     if "frames" not in writer.manifest:
         logging.info("No frames found in dataset manifest. Regenerating all frames.")
         return True
     else:
         # Check that all frames exist. If any are missing, frames should be regenerated.
         for frame in writer.manifest["frames"]:
-            if not os.path.exists(writer.outpath / frame):
+            if not os.path.exists(writer.outpath + "/" + frame):
```

Comment:
`writer.outpath` is always a string, and using `/` is only available for `pathlib.Path`.

Reply:
then you could convert to pathlib.Path to use the `/` operator in a crossplatform way here (nit / optional but I just had to say it)

Reply:
Oh, the cross-platform argument is a good one... I'll make this change.